src/app/components/auth/login/login.component.spec.ts

```diff
@@ -18,18 +19,18 @@ describe('LoginComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ LoginComponent, InputComponent ],
+      declarations: [LoginComponent, InputComponent],
       providers: [
         GlobalService,
         AuthService,
         WindowService,
         ApiService,
         EndpointsService
       ],
-      imports: [ RouterTestingModule, HttpClientModule ],
-      schemas: [ NO_ERRORS_SCHEMA ]
+      imports: [RouterTestingModule, HttpClientModule, FormsModule],
+      schemas: [NO_ERRORS_SCHEMA]
     })
-    .compileComponents();
+      .compileComponents();
```

Comment:
In the future, here, we should definitely add test cases. After completing the pending PR's, please start opening PR's for adding test cases. (cc: @RishabhJain2018)